site/docs/guides/reverse-proxy.md

```diff
@@ -0,0 +1,167 @@
+# Nessie behind a Reverse Proxy
+
+If you run Nessie behind a Reverse Proxy like istio or nginx, both the reverse proxy and Nessie/Quarkus need to be
+configured accordingly.
+
+!!! warn
+    Make sure to understand what the configuration options mean, before you apply those to any production setup!
+
+!!! info
+    Passing the `X-Forwarded-Proto`, `X-Forwarded-Host` and `X-Forwarded-Port` headers to Nessie works for
+    Iceberg REST.
+
+The follow config options are mentioned only for documentation purposes. Consult the
+[Quarkus documentation](https://quarkus.io/guides/http-reference#reverse-proxy) for "Running behind a reverse proxy"
+and configure those depending on your actual needs.
+
+```properties
+quarkus.http.proxy.proxy-address-forwarding=true
+quarkus.http.proxy.allow-x-forwarded=true
+quarkus.http.proxy.enable-forwarded-host=true
+quarkus.http.proxy.enable-forwarded-prefix=true
+```
+
+!!! warn
+    Do NOT enable the above options unless your reverse proxy (for example istio or nginx)
+    is properly setup to set these headers but also filter those from incoming requests.
+
+!!! info
+    The newer `Forwarded` header is the newer version of the older `X-Forwarded-For`, `X-Forwarded-Host`,
+    `X-Forwarded-Proto` headers, but does _not_ provide a replacement for the `X-Forwarded-Prefix` header,
+    which is needed in some scenarios.
```

Comment:
There's no replacement for `X-Forwarded-Port` too, AFAIK. In a way, the port is more important. With TLS termination, a difference between external and internal port numbers is unavoidable, I think.

Reply:
The `by` directive of the `Forwarded` header seems like it could be used to propagate the external port information to the server, but it does not appear to be interpreted by Quarkus [code](https://github.com/quarkusio/quarkus/blob/cdc1637bebb289320c36c2c859691746751e65c8/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ForwardedParser.java#L129) ATM.

I believe it would be worth mentioning that `X-Forwarded-Port` is preferred from the Nessie perspective to avoid surprises on the users' side.

I might be missing something but then again our examples use onfy `X-Forwarded-*` headers.